build-tasks/markup.js

```diff
@@ -15,7 +20,7 @@ var gulp      = require('gulp'),
   compile = function() {
     return gulp.src(src.docs)
       .pipe(plugins.plumber())
-      .pipe(env.deploy ? plugins.pug(): plugins.pug(opts.pug))
+      .pipe(turnOffPrettyPrint())
```

Comment:
Personally, I would say a function would be a little overkill for this. Also, the naming wouldn't be right. `turnOffPrettyPrint` would imply that the function `always` turns off pretty printing but it doesn't. It's only a small check that is only specific to if you decide to use `pug` for markup compilation, so a one-liner will do as most people are likely to scrap the markup compilation anyhow.

Just the line
```js
if (env.deploy && opts.pug.pretty) opts.pug.pretty = false;
```
at the top of the `compile` function will do 👍